src/actions/dapps/makeBalancesAction.js

```diff
@@ -0,0 +1,10 @@
+import { createActions } from 'spunky';
+
+import getBalances from '../../util/getBalances';
+
+export const ID = 'balances_api';
+
+export default createActions(ID, ({ net, address, tokens = [], addedAddress } = {}) =>
```

Comment:
The structure of this doesn't match the other action-makers under `actions/dapps`.  This should be wrapped in a function that takes a `sessionId` and `requestId`, which then builds a unique identifier for the redux store. i.e.:

```js
export const ID = 'balances';

export default function makeBalancesActions(sessionId, requestId) {
  const id = generateDAppActionId(sessionId, `${ID}-${requestId}`);

  return createActions(id, ({ net, address, tokens = [] }) => () => {
    return getBalances({ net, address, tokens })
  });
}
```